### PR TITLE
Add VMware workstation based RackHD + InfraSIM integration in howto c…

### PR DESCRIPTION
**Pull request by XiaowenJiang**
1. Modify BMC binding use case in configuration chapter.
2. Add RackHD+InfraSIM practice instruction in howto chapter.

### CODE REVIEW

**Review by fub2 — COMMENTED** [review]
It should require operations to allow promiscuous mode and virtualization nesting. ESXi part highlights this. Is there any workstation related instruct ructions being added?

**Comment by XiaowenJiang** [comment]
@fub2 There is no vswitch/port group promisc mode need to be set in VMWare workstation.
I just added additional instruction to remind user of VT-d flag and promisc mode for nics.

File: docs/how_to.rst [hunk 1]
```diff
@@ -195,12 +195,92 @@ The following picture shows the deployment model for the integration of InfraSIM
...
+                -
+                    network_mode: bridge
+                    network_name: <bridge_name>
+                    device: e1000
+        ...
+        bmc:
+            interface: <BMC ifname>
+        ...
+
+#. Start RackHD service and InfraSIM service.
```
Comment (turtle-fly) [inline docs/how_to.rst:293]: Introduce hyperlink to RackHD quick start and InfraSIM quick start.

File: docs/how_to.rst [hunk 2]
```diff
@@ -195,12 +195,92 @@ The following picture shows the deployment model for the integration of InfraSIM
...
+        ...
+
+#. Install RackHD from source code.
+   Please refer RackHD document(http://rackhd.readthedocs.io/en/latest/rackhd/ubuntu_source_installation.html) to setup the RackHD Server.
+
+#. Install InfraSIM from source code.
+
+    * Please refer ``chapter3.2`` in this document(http://infrasim.readthedocs.io/en/docs-infrasim-compute/installation.html) to install InfraSIM.
+
+    * Modify yaml file to add BMC binding and change qemu network interface to bridge mode::
```
Comment (turtle-fly) [inline docs/how_to.rst:278]: Yaml file default location

File: docs/how_to.rst [hunk 3]
```diff
@@ -195,12 +195,92 @@ The following picture shows the deployment model for the integration of InfraSIM
...
     * "Control Network" is used to control the virtual servers.
 
-#. Deploy virtual compute node and virtual PDU on ESXi with the network connection as in the pictures show. The ESXi show have network connection with the RackHD server.
+    A. In virtual machine settings, create network adapters:
+
+    .. image:: _static/vmworkstation_rackhd_networking.png
+               :height: 400
+               :align: center
+
+    B. In ``/etc/network/interfaces``, configure the ``CONTROL`` network interface to static::
```
Comment (turtle-fly) [inline docs/how_to.rst:260]: ``CONTROL`` to ``<CONTROL ifname>``

Add a hint for user to check ifname.

File: docs/how_to.rst [hunk 4]
```diff
@@ -195,12 +195,92 @@ The following picture shows the deployment model for the integration of InfraSIM
...
+
+#. Configure network connection for InfraSIM virtual machine:
+
+    A. In virtual machine settings, create network adapters:
+
+        .. image:: _static/vmworkstation_infrasim_networking.png
+                   :height: 400
+                   :align: center
+
+    B. Modify ``/etc/network/interfaces`` for the ``BMC`` and ``CONTROL`` networks::
```
Comment (turtle-fly) [inline docs/how_to.rst:230]: Hint to check ifname

File: docs/how_to.rst [hunk 5]
```diff
@@ -195,12 +195,92 @@ The following picture shows the deployment model for the integration of InfraSIM
...
+           :align: center
+
+
+Please follow below steps to setup the entire environment. After that, RackHD can discover and manage the virtual server and virtual PDU just as the real physical server and PDU. Note that in the example, virtual machines are spinned up on VMWare workstation. Configuration is similiar on other platforms.
+
+#. Enable VT-d in BIOS on Physical Server as in chapter 11.1.
+
+#. Create two virtual machines(for InfraSIM/RackHD respectively) inside VMWare workstation, or use ova images built by InfraSIM and RackHD groups.
+
+#. Create a Custom network(name it ``VMnet2`` here) in VMWare workstation with configuration below:
```
Comment (turtle-fly) [inline docs/how_to.rst:214]: Emphasize it's `host-only` and explain something

File: docs/how_to.rst [hunk 6]
```diff
@@ -195,12 +195,92 @@ The following picture shows the deployment model for the integration of InfraSIM
...
+The networking connection between InfraSIM and RackHD is shown below:
+
+.. image:: _static/vmworkstation_infrasim_rackhd_network_connection.png
+           :height: 600
+           :align: center
+
+
+Please follow below steps to setup the entire environment. After that, RackHD can discover and manage the virtual server and virtual PDU just as the real physical server and PDU. Note that in the example, virtual machines are spinned up on VMWare workstation. Configuration is similiar on other platforms.
+
+#. Enable VT-d in BIOS on Physical Server as in chapter 11.1.
```
Comment (turtle-fly) [inline docs/how_to.rst]: Add internal hyperlink, \`text <page#mark>\`_

**Review by turtle-fly — CHANGES_REQUESTED** [review]

**Comment by XiaowenJiang** [comment]
@turtle-fly Modify docs as requests. Please help me review the changes

**Review by turtle-fly — APPROVED** [review]

**Pull request merged**